srcpkgs/vault/files/vault/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec vault server -config=/etc/vault.hcl
```

Comment:
I'd rather not have this running as root.  Please create a system user and run it as that instead.